src/sidebar/components/test/annotation-test.js

```diff
@@ -18,6 +18,16 @@ var fakeDocumentMeta = {
 };
 
 /**
+ * Return an `Error` object matching the type of error produced by failed API
+ * calls.
+ */
+function apiError(status, message) {
```

Comment:
>  that function looks like it's both defining what a store error looks like, and translating an API response into a store error.

This is true, but given that a store error is currently "an Error object with an Angular API response object as the `response` property" there isn't much point making a distinction IMO. There is no abstraction of the type of the response for example.